packages/host/app/components/operator-mode.gts

```diff
@@ -82,14 +92,21 @@ export default class OperatorMode extends Component<Signature> {
     (globalThis as any)._CARDSTACK_CARD_SEARCH = this;
     registerDestructor(this, () => {
       delete (globalThis as any)._CARDSTACK_CARD_SEARCH;
+      this.operatorModeStateService.clearStack();
     });
 
-    this.stack = new TrackedArray([
-      {
+    // afterRender to prevent recomputation errors
+    schedule('afterRender', () => {
+      this.addToStack({
         card: this.args.firstCardInStack,
         format: 'isolated',
-      },
-    ]);
+      });
+    });
+  }
+
+  get stack() {
+    // We return the first one until we start supporting 2 stacks
+    return this.operatorModeStateService.state.stacks[0].items;
```

Comment:
Later when we will support the 2nd stack I think we'll want to refactor this so that we have a `Stack` component.